### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by Sumitwarrior7**
### DOD

- [X] I have made small correction/improvements

### Changes / Instructions

_Updated Readme_

### CODE REVIEW

**Comment by Sumitwarrior7** [comment]
@Ebazhanov Sir you can merge this pr, cuz now it doesn't have any conflicts

**Comment by Sumitwarrior7** [comment]
@Ebazhanov Sir you can merge this pr, cuz now it doesn't have any conflicts

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
🤔 🚬

File: css/css-quiz.md [hunk 1]
```diff
@@ -2549,158 +2549,162 @@ input.invalid {
...
+                                                                                                          ```````````````
 
-                                                                                                      ```````````````
+                                                                                                      ````````````````
 
-                                                                                                  ````````````````
+                                                                                                  `````````````````
 
-                                                                                              `````````````````
+                                                                                              ``````````````````
```
Comment (Ebazhanov) [inline css/css-quiz.md:2663]: if possible please remove this inconsistence, i mean this bulshit ```````````````````` .

thank you in advance!

**Comment by Sumitwarrior7** [comment]
Conflicting CSS file has been resolved now, you can check and merge

**Review by Ebazhanov — APPROVED** [review]
Thanks a lot, great job @Sumitwarrior7 👍

**Pull request merged**